crates/mysten-metrics/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 
 use axum::{extract::Extension, http::StatusCode, routing::get, Router};
 use dashmap::DashMap;
+use parking_lot::Mutex;
```

Comment:
its slowly becoming the default in sui-core because its faster when there's no contention, which is usually the case, and you don't have to do `unwrap()` everytime. That said, none of that matters here and I could switch back if you prefer.

Reply:
I thought that the stdlib one was the faster version these days. It's been a bit since I looked into this though. No reason to change form what you have here